src/alert/demos/enUS/index.demo-entry.md

```diff
@@ -15,17 +15,17 @@ no-icon
 
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
-| closable | `boolean` | `false` |  |
-| show-icon | `boolean` | `true` |  |
-| title | `string` | `undefined` |  |
-| type | `'default' \| 'info' \| 'success' \| 'warning' \| 'error'` | `'default'` |  |
-| on-after-leave | `Function` | `undefined` |  |
-| on-close | `() => boolean \| Promise<boolean> \| any` | `() => true` |  |
+| closable | `boolean` | `false` | Can the alert information be turned off. |
+| show-icon | `boolean` | `true` | Whether to show the icon of alert. |
+| title | `string` | `undefined` | Title information of alert. |
+| type | `'default' \| 'info' \| 'success' \| 'warning' \| 'error'` | `'default'` | alert type. |
+| on-after-leave | `Function` | `undefined` | Callback function executed when the alert disappears. |
+| on-close | `() => boolean \| Promise<boolean> \| any` | `() => true` | The callback function executed when the close icon is clicked. |
 
 ## Slots
 
-| Name    | Parameters | Description |
-| ------- | ---------- | ----------- |
-| default | `()`       |             |
-| header  | `()`       |             |
-| icon    | `()`       |             |
+| Name    | Parameters | Description                                   |
+| ------- | ---------- | --------------------------------------------- |
+| default | `()`       | The content of the alert.      |
+| header  | `()`       | The content filled in the alert header.       |
+| icon    | `()`       | The content filled in the icon part of alert. |
```

Comment:
Like the comment above, and also a bit of rephrasing, since the original message is somewhat ambiguous IMO. I think a comment like: _Icon displayed in the alert_ is clearer.